server/controllers/refer/referralsController.ts

```diff
@@ -57,6 +57,26 @@ export default class ReferralsController {
     }
   }
 
+  complete(): TypedRequestHandler<Request, Response> {
+    return async (req: Request, res: Response) => {
+      TypeUtils.assertHasUser(req)
+
+      const { referralId } = req.params
+
+      const referral = await this.referralService.getReferral(req.user.token, referralId)
+
+      if (referral.status !== 'referral_submitted') {
```

Comment:
Can either do this or redirect to task list

Reply:
I think a 400 here makes sense - it's unlikely they'll get here in a usual user journey so throwing an error feels appropriate.